src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,100 @@
-export const Pagination = () => {};
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
```

Comment:
what does it means?Tr to use better naming

Reply:
in readme.md was suggested bad naming

    <Pagination
      total={42} // total number of items to paginate
      perPage={5} // number of items per page
      currentPage={1} /* optional with 1 by default */
      onPageChange={(page) => { ... }}
    />